src/Readers/ODataReader.v4/OdcmReader.cs

```diff
@@ -117,6 +118,16 @@ private void AddVocabularyAnnotations(OdcmObject odcmObject, IEdmVocabularyAnnot
                 odcmObject.Annotations = ODataVocabularyReader.GetOdcmAnnotations(_edmModel, annotatableEdmEntity).ToList();
                 odcmObject.Description = _edmModel.GetDescriptionAnnotation(annotatableEdmEntity);
                 odcmObject.LongDescription = _edmModel.GetLongDescriptionAnnotation(annotatableEdmEntity);
+
+                if(annotatableEdmEntity is IEdmEntitySet && odcmObject is OdcmProperty)
+                {
+                    OdcmCapability.SetCapabilitiesForEntitySet((OdcmProperty)odcmObject);
+                }
+
+                if (annotatableEdmEntity is IEdmEntityContainer && odcmObject is OdcmServiceClass)
+                {
+                    OdcmCapability.SetCapabilitiesForEntityContainer((OdcmServiceClass)odcmObject);

```

Comment:
Same here. To clarify, I think this is important because we currently have the responsibility of setting the OdcmProjection of an OdcmProperty split amongst two  actors which means future edits will require remembering that we need to think about both.